packages/client/modules/outcomeCard/containers/OutcomeCard/OutcomeCardContainer.tsx

```diff
@@ -100,14 +101,15 @@ const OutcomeCardContainer = memo((props: Props) => {
       UpdateTaskMutation(atmosphere, {updatedTask, area}, {})
     }
   }
-
   useScrollIntoView(ref, !contentState.hasText())
+  useClickAway(ref, () => setIsTaskHovered(false))
   return (
     <Wrapper
       tabIndex={-1}
       className={className}
       onMouseEnter={() => setIsTaskHovered(true)}
       onMouseLeave={() => setIsTaskHovered(false)}
+      onMouseOver={() => setIsTaskHovered(true)}
```

Comment:
+1 what's the reason for adding this? I tested and everything seems to work correctly with just using `useClickAway`

Reply:
@BartoszJarocki here's the explanation for this

Behavior Before implementation "Onmouseover is not added"
https://www.loom.com/share/8716b3e52f9e43da9d9916ec804f1290

After implementation "Onmouseover is added"
https://www.loom.com/share/42f0c35fb07049ed9c3f624b070a8bd6

Reply:
@BartoszJarocki here's the explanation for this

Behavior Before implementation "Onmouseover is not added"
https://www.loom.com/share/8716b3e52f9e43da9d9916ec804f1290

After implementation "Onmouseover is added"
https://www.loom.com/share/42f0c35fb07049ed9c3f624b070a8bd6